runtime/centrifuge/src/migrations.rs

```diff
@@ -10,4 +10,116 @@
 // MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 // GNU General Public License for more details.
 
-pub type UpgradeCentrifuge1024 = ();
+pub type UpgradeCentrifuge1025 = burn_unburned::Migration<super::Runtime>;
+
+// Copyright 2021 Centrifuge Foundation (centrifuge.io).
+//
+// This file is part of the Centrifuge chain project.
+// Centrifuge is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version (see http://www.gnu.org/licenses).
+// Centrifuge is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+mod burn_unburned {
+	const LOG_PREFIX: &str = "BurnUnburnedMigration: ";
+	const LP_ETH_USDC: CurrencyId = CurrencyId::ForeignAsset(100_001);
```

Comment:
Out of curiosity where `100_001` comes from?

Reply:
It's the LP ETH USDC asset in centrifuge mainnet. 
Query orml-asset-registry if curious